src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -265,8 +262,12 @@ public void updateSmartDashboard() {
 
   @Override
   public void periodic() {
+    smoothRPM =
+        Constants.Shooter.kA * flywheelEncoder.getVelocity()
```

Comment:
is RelativeEncoder.getVelocity() a hardware call?

If so, does it make sense to get it once, and use it for smoothRPM and sending to the dashboard?